api/src/test/java/org/pmiops/workbench/leonardo/LeonardoApiClientTest.java

```diff
@@ -303,6 +309,18 @@ public void testDeleteAppSuccess() throws Exception {
     verify(userAppsApi).deleteApp(GOOGLE_PROJECT_ID, appName, deleteDisk);
   }
 
+  @Test
+  public void testRuntimeConfigDeserializer() throws Exception {
```

Comment:
what is the purpose of this test?

Reply:
just for testing the generated client can deserialize the payload from Leonardo properly